src/frontend/src/components/AboutFooter/AboutFooter.jsx

```diff
@@ -51,6 +52,9 @@ const AboutFooter = () => {
   const classes = useStyles();
   const theme = useTheme();
   const matches = useMediaQuery(theme.breakpoints.up('md'));
+  const { contributor, loading, error } = useTelescopeContributor();
+
+  console.log(contributor, loading, error);
```

Comment:
Get rid of this.

Reply:
I left the console.log in so that the person who will display it can easily see the shape of the object to manipulate. I was planning on adding a note in the issue to remove the console.log after they found out what attributes they wanted from it.

Let me know if you still want me to remove it. Thanks.

Reply:
Yeah, we never leave console.log statements in production code.